rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -325,7 +325,7 @@ namespace rsx
 	{
 		//Endianness is swapped because common upload code expects input in BE
 		//TODO: Implement fast upload path for LE inputs and do away with this
-		element_push_buffer.push_back(se_storage<u32>::swap(index));
+		element_push_buffer.push_back(be_t<u32>{index}.raw());
```

Comment:
Need this swapped for now

Reply:
It is swapped, I'm using be_t for it to obey IS_BE_MACHINE macro.

Reply:
Doesn't .raw() just return index with the default endianness? Or maybe I'm misunderstanding its use?

Reply:
The constructor *and* the converter byteswap, so by using raw() I'm making it only byteswap at the constructor, effectively.